WordPress/Classes/Models/ReaderPost.m

```diff
@@ -67,11 +68,7 @@ - (BOOL)isCrossPost
 
 - (BOOL)isAtomic
 {
-    // TODO: This is temporary until we start parsing a new value from the reader endpoint.
-    //
-    // Issue: https://git.io/JvNAN
-    //
-    return false;
+    return self.isBlogAtomic;
```

Comment:
Would it make sense to update `ReaderPostContentProvider.h` to change `isAtomic` to `isBlogAtomic` and then update `MediaHost+ReaderPostContentProvider.swift` accordingly and then remove this `isAtomic` method completely?

Reply:
+1 on this one!